rest-api-spec/src/main/resources/rest-api-spec/test/search/340_doc_values_field.yml

```diff
@@ -449,6 +449,28 @@
 
   - match: { hits.total: 2 }
 
+  - do:
```

Comment:
We may need to apply for term search over the network range

```
  - skip:
      version: " - 2.18.99"
      reason: "fixed in 2.19.0"
```

(I think it does not work now, right?)

Reply:
Perhaps. Pushed it removed.

Reply:
turns out notabest idea 
https://build.ci.opensearch.org/job/gradle-check/50720/testReport/junit/org.opensearch.backwards/MixedClusterClientYamlTestSuiteIT/test__p0_search_340_doc_values_field_search_on_fields_with_only_doc_values_enabled_/